LayoutTests/editing/selection/text-selection-counter.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<script>
+function selectRangeAfterLayoutAndPaint(startElement, startIndex, endElement, endIndex) {
```

Comment:
Looks like this function is never used?

Reply:
Yes - after removing, the test was working fine, so I removed it.